conf/pxe_cluster/hosts.yaml

```diff
@@ -50,6 +50,8 @@ PROVISION:
     ftp_proxy: ""
     http_proxy: "http://10.197.121.21:3142"
```

Comment:
This should URL should have never been populated.

Reply:
We can mention the following in hosts.yaml
http_proxy: ""

But then the onus lies on the end user to populate it with Build server IP : port.
This information is applied on apt.conf of all the nodes so that they can download and install the packages through this proxy

So should we keep it blank and update host.yaml + install.md

Reply:
Agree with Sachin to make http_proxy blank as well.